data/db-schema/src/main/resources/net/corda/db/schema/statemanager/migration/state-manager-migration-v5.1.xml

```diff
@@ -30,6 +30,9 @@
                 <constraints nullable="false"/>
             </column>
         </createTable>
+        <createIndex tableName="state" indexName="state_version_idx">
+            <column name="version"/>
+        </createIndex>
```

Comment:
Won't the index need to include (key, version) - particularly for these queries?
```
DELETE FROM $STATE_MANAGER_TABLE s WHERE s.key = :$KEY_PARAMETER_NAME AND s.version = :$VERSION_PARAMETER_NAME

UPDATE $STATE_MANAGER_TABLE SET
            key = :$KEY_PARAMETER_NAME, value = :$VALUE_PARAMETER_NAME, version = version + 1, metadata = CAST(:$METADATA_PARAMETER_NAME as JSONB), modified_time = CURRENT_TIMESTAMP AT TIME ZONE 'UTC'
            WHERE key = :$KEY_PARAMETER_NAME AND version = :$VERSION_PARAMETER_NAME
```

Reply:
Think your'e right... will update the changelog and open another `PR`.